backend/configs/usr/lib/cgi-bin/download_everything.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+run_pigz() {
+	# if there are more than 1 core, use half of them
+	cpu_cores=$(nproc --all)
+	if [ $cpu_cores -gt 1 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)